### PR TITLE
SW-5170 Select all orgs when attempting to find org name for project

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2484** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2484?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/AcceleratorRouter/ParticipantProjects/ParticipantProjectProvider.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const ParticipantProjectProvider = ({ children }: Props) => {
 
   useEffect(() => {
     if (project?.organizationId) {
-      const request = dispatch(requestAcceleratorOrgs({ locale: activeLocale }));
+      const request = dispatch(requestAcceleratorOrgs({ locale: activeLocale, includeParticipants: true }));
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/ParticipantProjects/ParticipantProjectProvider.tsx:80]: use `useAcceleratorOrgs`?

Comment (nickgraz) [reply src/scenes/AcceleratorRouter/ParticipantProjects/ParticipantProjectProvider.tsx]: 🙌

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**